app/views/applicants/new.html.erb

```diff
@@ -30,16 +30,23 @@
   <table  class="lined">
     <%= form_tag(new_project_applicant_path(project_id: params[:project_id]), method: :get, id: "username_form") do %>
       <tr>
-        <td class="title">PI Username</td>
+        <td class="title">
+          <%= "#{@project.applicant_abbreviation_wording.html_safe} Username" -%>
```

Comment:
`@project.applicant_abbreviation_wording.html_safe` could be extracted into a helper method

if `@project.applicant_abbreviation_wording` is nil - does html_safe still work? or do we get a NoMethodError on NilClass - `@project.applicant_abbreviation_wording&.html_safe` is probably safer (if we're on a modern Ruby version)